src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java

```diff
@@ -115,6 +227,10 @@
 
     private static ImmutableList<BlockBerryBush> allBerryBushBlocks;
 
+    private static ImmutableList<ItemBlock> allAlabasterRawBlocks;
+    private static ImmutableList<ItemBlock> allAlabasterBricksBlocks;
+    private static ImmutableList<ItemBlock> allAlabasterPolishedBlocks;
+
```

Comment:
These shouldn't be necessary, the item blocks are all simple items and can have their model registration bundled with everything else.
```suggestion
```